hydromt/model/components/vector.py

```diff
@@ -341,15 +341,19 @@ def write(
             )
         # write to geojson only
         elif filename is None:
-            os.makedirs(dirname(join(self.root.path, geometry_filename)), exist_ok=True)
+            full_path = join(self.root.path, geometry_filename)
+            os.makedirs(dirname(full_path), exist_ok=True)
             gdf = ds.vector.to_gdf(**kwargs)
-            gdf.to_file(join(self.root.path, geometry_filename))
+            logger.debug(f"Writing file {full_path}")
```

Comment:
I'm a little confused now about what we decided. I think it is good that the user sees staticmaps will be written to file ...nc. So the logger level should be info rather than debug no? Because the final path is done in core rather than the plugin, core should then be the place to print this info I think. Also good if the component name in model can be part of the message.